configs/machines/aleph.yaml

```diff
@@ -108,7 +182,7 @@ export_vars:
         # Important for eccodes to work
         CRAYPE_LINK_TYPE: dynamic
 
-label_format: '"[@MODEL@]%l:"'
+label_format: '"[@MODEL@]%r:"'
```

Comment:
What does this change do?

Reply:
This makes it so that PBS outputs the global MPI task number in front of every logfile output line instead of the local one. This matches the behavior of slurm and is generally more useful for debugging.